### PR TITLE
Base path fix

### PR DESCRIPTION
**Pull request by rjnrohit**
map file location relative to current working directory
- [X] Sem Tests Pass

### CODE REVIEW

**Comment by swapnilraj** [comment]
@GuyMcComb can we run the semantic tests as well

File: src/index.ts [hunk 1]
```diff
@@ -94,7 +94,12 @@ program
       options.basePath = options.basePath || defaultBasePath;
     }
 
-    const ast = compileSolFiles(files, options);
+    // map file location relative to current working directory
+    const mFiles = files.map((file) => {
+      return path.relative(process.cwd(), file);
+    });
```
Comment (piwonskp) [inline src/index.ts]: Can be shortened by removing brackets and return
```suggestion
    const mFiles = files.map(file => path.relative(process.cwd(), file));
```

Comment (rjnrohit) [reply src/index.ts]: great, I'll do

**Review by piwonskp — APPROVED** [review]

File: tests/behaviour/test_calldata.ts [hunk 2]
```diff
@@ -5165,7 +5375,7 @@ const tests: { [file: string]: ITestCalldata[] | null } = {
           '0x000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000003',
         expectations: '0x',
         failure: false,
-        signature: 'constructor(uint256[3])',
```
Comment (GuyMcComb) [inline tests/behaviour/test_calldata.ts]: Whats going on here @RohitRanjangit

Comment (rjnrohit) [reply tests/behaviour/test_calldata.ts]: Nothing

**Review by GuyMcComb — APPROVED** [review]

**Pull request merged**